app/controllers/cim_instance_controller.rb

```diff
@@ -147,7 +147,7 @@ def process_show(associations = {})
     else
       whitelisted_key = associations.keys.find { |key| key == @display }
       if whitelisted_key.present?
-        model_name = whitelisted_key.singularize.classify.constantize
+        model_name = whitelisted_key.singularize.classify
         drop_breadcrumb( {:name=>@record.evm_display_name+" (All #{ui_lookup(:tables => @display.singularize)})", :url=>"/#{self.class.table_name}/show/#{@record.id}?display=#{@display}"} )
```

Comment:
This use of `singularize` appears to conflict with the nearby use of `self.class.table_name` (which is plural). @martinpovolny which is right? I'm guessing plural / the table name?